.github/workflows/release-docker-images.yml

```diff
@@ -0,0 +1,48 @@
+name: Monstache release docker images
+
+on:
+  release:
+    types: [created]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+permissions:
+    contents: write
```

Comment:
Is this permission needed here since this is pushing to docker Hub?  My understanding is that this is needed to write to Github?

Reply:
Probably not. Will remove it.